lib/native.js

```diff
@@ -5,6 +5,11 @@ var promisify = require('util').promisify;
 var resolveTxt = promisify(require('dns').resolveTxt);
 var crypto = require('crypto');
 
+const https = require('https');
+var http01Agent = new https.Agent({
+	rejectUnauthorized: false
```

Comment:
This should be an option or environment variable. The default behavior should not be to allow unsigned, expired, and/or unverifiable certificates.